package.json

```diff
@@ -16,26 +17,30 @@
     "test": "tests"
   },
   "scripts": {
-    "test": "gulp test",
-    "prepublish": "gulp build",
+    "test:node": "mocha --reporter min tests/**/*.spec.js",
+    "test:browser": "karma start",
+    "test": "tslint lib/**/*.ts && npm run build && npm run test:node && npm run test:browser",
+    "build": "rimraf ./build && npm run prettify:src && tsc && npm run prettify:dist",
+    "prepublish": "npm run build && npm run readme && npm run test",
+    "prettify:src": "prettier --use-tabs --single-quote --write lib/*.ts",
+    "prettify:dist": "prettier --use-tabs --single-quote --write build/*.js",
```

Comment:
Why do you run prettifier on the generated code? It'd be better imo to leave it as is (rather than risk messing up the sourcemaps)